site/content/docs/testing/_index.md

```diff
@@ -33,20 +33,30 @@ in your [test sessions page](https://app.checklyhq.com/test-sessions).
 
 ![test sessions overview](/docs/images/testing/test_session_overview@2x.jpg)
 
-The test session overview gives you insights into where a test session was triggered from, who triggered it and optionally
-shows git branch and commit information.
-
-For each test session, we record all logging, videos, traces, screenshots and other telemetry. This specifically powerful
-when using our `@playwright/test` powered browser checks.
+The test session overview provides insights into where a test session was triggered from and who triggered it.
 
 ![test sessions detail](/docs/images/testing/test_session_detail@2x.jpg)
 
+While test session details are automatically detected you can also configure them:
+
+| Test session detail | Source                                                                       |
```

Comment:
Maybe DRY up this table a bit following the `{{< markdownpartial "/_shared/cli-basics-for-cicd.md" >}}` snippet we use in the CI/CD integration docs? I'm afraid we will create drift sprinkling these sections throughout the docs.